src/test/java/seedu/address/model/person/AllergyTest.java

```diff
@@ -17,10 +17,4 @@ public void constructor_invalidTagName_throwsIllegalArgumentException() {
         assertThrows(IllegalArgumentException.class, () -> new Allergy(invalidTagName));
     }
 
-    @Test
-    public void isValidTagName() {
-        // null tag name
-        assertThrows(NullPointerException.class, () -> Allergy.isValidAllergyName(null));
-    }
-
```

Comment:
You need to test `isValidAllergyName` separately because it is a `public static` method that can be used without the `Allergy()` constructor.